src/swagger.php

```diff
@@ -0,0 +1,15 @@
+<?php
```

Comment:
Wasn't entirely sure where to put it, it's all the Swagger required information, then the tags e.g. release, toggles, for all the groups of api stuff. But again, not entirely sure where to put it